app/controllers/admin/topics_controller.rb

```diff
@@ -202,7 +202,7 @@ def assign_agent
     if params[:topic_ids].count > 1
       get_tickets
     else
-      @topic = @topics.first
+      @topic = Topic.find(@topics.first.id)
```

Comment:
That's an interesting bug. Although I didn't consider it before, it makes sense that the `@topics.first` has been modified by a direct query so ActiveRecord is not able to update the in-memory version.

A neater way to achieve this (although it does the same thing under the hood) is to use

```
@topic = @topics.first.reload
```